README.md

```diff
@@ -9,17 +9,35 @@ Discussions for AI conformance requirements are now tracked using the
 Unresolved items will be discussed during the
 [recurring WG meeting](https://github.com/kubernetes/community/tree/master/wg-ai-conformance#meetings).
 
+### Process Details
+
+This process adopts the Kubernetes Enhancement Proposal (KEP) process as the basis for managing the lifecycle of Kubernetes AI conformance Requirements (KARs), including review, discussion, and approval.
+
+- **Requirement Proposal**: Propose each new Kubernetes AI conformance Requirement (KAR) as a GitHub issue. Create a PR for the KAR targeting a specific Kubernetes release. Progress for each requirement will be tracked using the GitHub issue and the KAR. Each requirement will be reviewed individually.
+- **Graduation Criteria**: Requirements will graduate from "SHOULD" to "MUST" stage in alignment with the process used for KEPs in each Kubernetes release cycle.
+- **Timeline Alignment**: The lifecycle for requirements will follow the Kubernetes release schedule: e.g. https://github.com/kubernetes/sig-release/blob/master/releases/release-1.35/README.md#timeline This timeline will be adopted starting Kubernetes v1.36.
+  - **KEP/KAR Freeze**: Locks in the set of KARs to be considered for updates for a given Kubernetes release. No new requirements after KEP freeze.
+  - **Discussion and Refinement before Code Freeze**: After KEP/KAR freeze, all discussions, text refinement, any changes (including associated tests) for accepted requirements will happen as part of the PR review for KAR updates. Before the code freeze deadline for the given Kubernetes release, a PR with all the AI conformance requirements for that Kubernetes release in the form of conformance-versions/KubernetesAIConformance-1.NN.yaml and all changes for all KARs for that Kubernetes release must be reviewed, approved, and merged. 
+  - **Post Code Freeze**: A PR with a copy of conformance-versions/KubernetesAIConformance-1.NN.yaml must be reviewed and merged in [cncf/ai-conformance](https://github.com/cncf/ai-conformance) to ensure transparency and clarity for the entire community. In the event a kubernetes feature does not reach GA and impacts the graduation of a KAR, we will need to reassess that KAR to rollback and update conformance-versions/KubernetesAIConformance-1.NN.yaml accordingly.
+- **Reviewers**: everyone in wg-ai-conformance
+- **Approvers**: wg-ai-conformance leads, SIG Arch leads (lazy consensus), and SIG Testing leads (for automated testing)
+- **Stage**: All requirements need to start with SHOULD and eventually graduate to MUST
+- **Requirement removal**: Criteria to remove a requirement is TBD
+
+
 ## Designing AI Conformance Tests
 
+Once there is community consensus on a requirement, the next step is to define how to verify it.
+For every MUST requirement, a corresponding test will be designed. Starting v1.37, automated tests are prerequisites for new MUSTs. This test design should be documented as part of the KAR and should be specific enough to be implemented in an automated fashion.
```

Comment:
I am interested in hearing the input from others, but I think every requirement SHOULD or MUST ideally has a proposed test as that makes it easier for everyone to prepare for when requirements become fixed.  In general, I like the idea of marking a SHOULD requirement to become a MUST to require no change to the underlying test, similar to how an existing test is promoted to conformance in k8s.

See: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#promoting-tests-to-conformance